helm/values.yaml

```diff
@@ -0,0 +1,360 @@
+            {
+              "channelConfigType": "WEBHOOK",
+              "url": "https://hooks.slack.com/services/abc",
+              "webhookFormat": "WEBHOOK_FORMAT_SLACK"
+            }
+          ]
+        }
+      ]
+  alertRulesConfig:
+    name: hypertrace-alert-engine-alert-rules-config
+    rules:
+      [
+        {
+          "id": "notification_rule_1",
+          "ruleName": "high_avg_latency",
+          "description": "Alert for high avg latency of payment service",
+          "eventConditionId": "event_condition_1",
+          "eventConditionType": "metricAnomalyEventCondition",
+          "channelId": "channel-id-1",
+          "eventCondition": {
+            "metricSelection": {
+              "metricAttribute": {
+                "key": "duration",
+                "scope": "SERVICE"
+              },
+              "metricAggregationFunction": "METRIC_AGGREGATION_FUNCTION_TYPE_AVG",
+              "metricAggregationInterval": "PT15S",
+              "filter": {
+                "leafFilter": {
+                  "lhsExpression": {
+                    "attribute": {
+                      "key": "name",
+                      "scope": "SERVICE"
+                    }
+                  },
+                  "valueOperator": "VALUE_OPERATOR_EQ",
+                  "rhsExpression": {
+                    "stringValue": "customer"
+                  }
+                }
+              }
+            },
+            "violationCondition": [ {
+              "staticThresholdCondition": {
+                "operator": "STATIC_THRESHOLD_OPERATOR_GT",
+                "value": 5.0,
+                "minimumViolationDuration": "PT5M",
+                "severity": "SEVERITY_CRITICAL"
+              }
+            } ]
+          }
+        }
+      ]
+
+metricAnomalyDetectorConfig:
+  name: metric-anomlay-detector-config
+  replicaCount: 1
+  maxUnavailable: 0
+  image:
+    repository: hypertrace/metric-anomlay-detector
+    pullPolicy: IfNotPresent
+    tagOverride: ""
+  imagePullSecrets: [ ]
+  nodeLabels: { }
+  containerAdminPort: 8099
+  javaOpts: "-XX:InitialRAMPercentage=50.0 -XX:MaxRAMPercentage=75.0 -XX:MaxDirectMemorySize=128M"
+  livenessProbe:
+    initialDelaySeconds: 10
+    periodSeconds: 5
+  readinessProbe:
+    initialDelaySeconds: 10
+    periodSeconds: 15
+    failureThreshold: 5
+  deploymentLabels:
+    app: metric-anomlay-detector
+  podLabels:
+    app: metric-anomlay-detector
+  podAnnotations: { }
+  deploymentSelectorMatchLabels:
+    app: metric-anomlay-detector
+  serviceSelectorLabels:
+    app: metric-anomlay-detector
+  resources:
+    requests:
+      cpu: 0.1
+      memory: 1024Mi
+    limits:
+      cpu: 0.2
+      memory: 1024Mi
+  logConfig:
+    name: metric-anomlay-detector-log-appender-config
+    monitorInterval: 30
+    rootLogger:
+      level: INFO
+    appender:
+      rolling:
+        enabled: false
+
+  queryService:
+    host: query-service
+    port: 9012
```

Comment:
values should be based on the standalone ports

Reply:
done